README.md

```diff
@@ -89,6 +89,20 @@ for detailed examples and documentation of the individual decorators.
 Development
 ------------------------------------------------------------------------------
 
+### Specs
+
+Ember Decorators follows the [current decorators "stage 2" proposal specs](https://github.com/tc39/proposal-decorators).
```

Comment:
Should we mention the sha that we comply with?

Reply:
Interesting idea. Do you think this is maintainable?

Ultimately our compliance is up to the current state of Babel. Maybe it makes sense to link to instead / as well:

- [Spec: "Decorators"](https://github.com/babel/babel/pulls?utf8=%E2%9C%93&q=label%3A%22Spec%3A+Decorators%22)
- [Spec: "Class Fields"](https://github.com/babel/babel/pulls?utf8=%E2%9C%93&q=label%3A%22Spec%3A+Class+Fields%22)
- [`@ember-decorators/babel-transforms/package.json`](https://github.com/ember-decorators/ember-decorators/blob/master/packages/babel-transforms/package.json#L26-L27)